index.js

```diff
@@ -13,11 +18,50 @@ const path = require('path');
  * @param  {String|Array<String>} options.directory - the location(s) of all sass files
  * @return {String}
  */
-module.exports = function({ dependency, filename, directory } = {}) {
+module.exports = function({
+  dependency,
+  filename,
+  directory,
+  webpackConfig
+} = {}) {
   if (dependency === undefined) throw new Error('dependency is not supplied');
   if (filename === undefined) throw new Error('filename is not supplied');
   if (directory === undefined) throw new Error('directory is not supplied');
 
+  const WEBPACK_ALIAS_FLAG = '~';
+  if (dependency.startsWith(WEBPACK_ALIAS_FLAG) && Boolean(webpackConfig)) {
+    dependency = dependency.replace(WEBPACK_ALIAS_FLAG, '');
+    let loadedConfig;
+
+    try {
+      webpackConfig = path.resolve(webpackConfig);
+      loadedConfig = require(webpackConfig);
+
+      if (typeof loadedConfig === 'function') {
+        loadedConfig = loadedConfig();
+      } else if (Array.isArray(loadedConfig)) {
+        loadedConfig = loadedConfig[0];
+      }
+    } catch (error) {
+      debug(`error loading the webpack config at ${webpackConfig}`);
+      debug(error.message);
+      debug(error.stack);
+      return '';
```

Comment:
I was looking at this and I was wondering why aren't you just throwing the error?

Reply:
I would like to be consistent with other `lookup` behaviors, such as https://github.com/dependents/node-filing-cabinet/blob/main/index.js#L443